consensus/src/epoch_manager.rs

```diff
@@ -950,29 +961,68 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
         } else {
             let mut rng =
                 StdRng::from_rng(thread_rng()).map_err(NoRandomnessReason::RngCreationError)?;
-            let augmented_key_pair = WVUF::augment_key_pair(&vuf_pp, sk, pk, &mut rng);
+            let augmented_key_pair = WVUF::augment_key_pair(&vuf_pp, sk.main, pk.main, &mut rng);
+            let fast_augmented_key_pair = if fast_randomness_is_enabled {
+                Some(WVUF::augment_key_pair(
+                    &vuf_pp,
+                    sk.fast.unwrap(),
+                    pk.fast.unwrap(),
+                    &mut rng,
+                ))
+            } else {
+                None
+            };
             self.rand_storage
                 .save_key_pair_bytes(
                     new_epoch,
-                    bcs::to_bytes(&augmented_key_pair)
+                    bcs::to_bytes(&(augmented_key_pair.clone(), fast_augmented_key_pair.clone()))
```

Comment:
i wonder if it is safer to have a `self.fast_rand_storage`.

Reply:
I will leave it to real engs to decide (and fix ;))

Reply:
approved but let me think more about this.

Reply:
this is also a schema breaking change if we already enabled it

Reply:
Currently just randomnet? devnet is wiped regularly and is fine?